### PR TITLE
Fix couple of typos

### PR DESCRIPTION
**Pull request by KKSzymanowski**
Fixed three typos in the Readme.

### CODE REVIEW

**Comment by kubawerlos** [comment]
Nope.
> Do not modify README.rst manually!

from [CONTRIBUTING.md](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/CONTRIBUTING.md)

You have to fix these typos in fixers files and regenerate readme.

File: README.rst [hunk 1]
```diff
@@ -1401,7 +1401,7 @@ Where ``$COMMIT_RANGE`` is your range of commits, eg ``$TRAVIS_COMMIT_RANGE`` or
 Exit codes
 ----------
 
-Exit code is build using following bit flags:
+Exit code is built using following bit flags:
```
Comment (SpacePossum) [inline README.rst]: not sure, but isn't `build` the active form here, I'm not a native English speaker so I'm not sure

**Comment by SpacePossum** [comment]
Thanks for the PR!
:+1: for the fixes (please see one comment), but as @kubawerlos stated these typos should be fixed at the source and not in the README directly. Let me know if you've the time to do it, otherwise I can help out on this PR.

File: README.rst (hunk 1 above)
Comment (kubawerlos) [reply README.rst]: Me ~~too~~ neither, so I thought I comment ;)

You are right @SpacePossum - `build` is the active from, but after `is` don't we suppose to use passive form?

Comment (SpacePossum) [reply README.rst]: ah yes, looks like a good change to me :+1:

**Comment by KKSzymanowski** [comment]
Sorry, I haven't looked at the contributing guideline.
I'll open a new proper PR.

**Pull request closed without merging**